core/spec/spec_helper.rb

```diff
@@ -10,9 +10,6 @@
   SimpleCov.start('rails')
 end
 
-require 'spree/testing_support/buildkite_test_collector'
-Spree::TestingSupport::BuildkiteTestCollector.enable
```

Comment:
When/if we'll reinstate it we could try driving the call to enable with an environment variable, so temporary hiccups can be handled in the CircleCI settings…

Reply:
Good call 👍